src/utils/build-info.ts

```diff
@@ -122,3 +123,32 @@ command = "${chosenSettings.devCommand}"
     return chosenSettings
   }
 }
+
+/**
+ * Merges the settings from the heuristics with the cached config.
+ * Returns the updated cached config
+ * @param cachedConfig The cached config
```

Comment:
I think this needs updating, since I don't see this param?

Reply:
oh good catch!

Reply:
fixed here: https://github.com/netlify/cli/pull/6729/commits/049f4a9c2819f2326e2289c593613defca3ea828